common/failpoint/failpoint_test.go

```diff
@@ -12,37 +12,37 @@ import (
 	"testing"
 
 	"github.com/mongodb/mongo-tools/common/testtype"
-	. "github.com/smartystreets/goconvey/convey"
+	c "github.com/smartystreets/goconvey/convey"
 )
 
 func TestFailpointParsing(t *testing.T) {
 	testtype.SkipUnlessTestType(t, testtype.UnitTestType)
 
-	Convey("With test args", t, func() {
+	c.Convey("With test args", t, func() {
 		args := "foo=bar,baz,biz=,=a"
 		ParseFailpoints(args)
 
-		So(Enabled("foo"), ShouldBeTrue)
-		So(Enabled("baz"), ShouldBeTrue)
-		So(Enabled("biz"), ShouldBeTrue)
-		So(Enabled(""), ShouldBeTrue)
-		So(Enabled("bar"), ShouldBeFalse)
+		c.So(Enabled("foo"), c.ShouldBeTrue)
+		c.So(Enabled("baz"), c.ShouldBeTrue)
+		c.So(Enabled("biz"), c.ShouldBeTrue)
+		c.So(Enabled(""), c.ShouldBeTrue)
+		c.So(Enabled("bar"), c.ShouldBeFalse)
```

Comment:
There's a collision between the `Reset()` func declared in our `failpoint` package here and the `convey` package that we use for tests.

I'm not sure how that wasn't an issue before (maybe it had to do with the `// +build failpoints` build constraint at the top), but we should specify the `convey` funcs with an alias for this test file anyway.